### DISCUSSION TITLE
[Solution] One of the scenarios where the monitor can't adjust the brightness

### DISCUSSION THREAD

**Discussion by XMuli** [xanderfrangos/twinkle-tray#804] [General]
After reading [Common reasons for monitors not being supported or detected]( https://github.com/xanderfrangos/twinkle-tray/wiki/Display-Detection-&-Support-Issues#common-reasons-for-monitors-not-being-supported-or-detected) thought I could add one.

It was found that monitors in one of the `Color Gamut & Color Temperature` modes would not be able to adjust the brightness using 
twinkle-tray; switching to another mode would do the trick. **Suggested adding to the wiki**

![image](https://github.com/user-attachments/assets/b02e87f5-2055-455c-a195-bc012840f56d)

**Comment by xanderfrangos** [comment]
Thanks for the tip! I'll add this to the wiki when I can.
